### ISSUE TITLE
Поддерживается ли поиск с возможными заменами е<>ё?

### ISSUE THREAD

**Issue by DenisNP** [bzaar/DawgSharp#21]
В реализации DAWG на Python есть поиск с заменами, который позволяет находить слова с **ё** даже если записать их через **е**.  Есть ли здесь такое?

**Comment by bzaar** [comment]
Нет, сейчас библиотека не учитывает особенности какого-либо конкретного
языка. Достичь желаемого эффекта можно,  например, заменой ё на е перед
добавлением в словарь и пепед поиском в нем.

**Comment by bzaar** [comment]
Теперь поиск с заменами поддерживается. [Пример](https://github.com/bzaar/DawgSharp/commit/291156c27922b813d7e5b98dc50eac0a94ba7cda#diff-e67558064f5365635c95aa7342bc84a0dc5406a785f4c86e5071fc8026c45651R10-R35).
